rtribbia/cards3-template.js

```diff
@@ -0,0 +1,136 @@
+function isInt(n) { return ((n%1 === 0) && (typeof n === 'number')); }
```

Comment:
Good work throughout! Looks like you've got this pretty much handled-just one comment for you in the `alphaSort` method below.